dualStepperDPTStartWithRadar.ino

```diff
@@ -122,6 +122,17 @@ void refreshTargets() {
 
 			auto newPositionObservation = PositionVector(fixed(target->x) / 1000, fixed(target->y) / 1000, fixed(target->z) / 1000);
 			dptState.updateTargetById(target->id, target->valid, newPositionObservation, 8);
+			break;
+		case cerializer::Config::Type():
+			auto config = static_cast<cerializer::Config*>(thing.get());
+			dptState.config.projectile_speed = fixed(config->projectile_speed);
+			dptState.config.turret_height = fixed(config->turret_height);
+			dptState.config.gravity = fixed(config->gravity);
+			dptState.stepperA.setMaxSpeed(config->max_speed);
+			dptState.stepperA.setAcceleration(config->acceleration);
+			dptState.stepperB.setMaxSpeed(config->max_speed);
+			dptState.stepperB.setAcceleration(config->acceleration);
```

Comment:
Move this logic into a method that takes a config message and then updates the state.